hokusai/production.yml

```diff
@@ -63,6 +63,78 @@ spec:
                 values:
                 - foreground
 
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metaphysics-web-green
```

Comment:
Besides this name change, setting `replicas: 2` and the env overrides below, this is identical to the existing metaphysics-web deployment